src/components/GWL.vue

```diff
@@ -190,189 +208,273 @@ export default {
-        .attr("id", "x-line")
-
-      // scale space
-      var xScale = this.d3.scaleLinear()
-        .domain([0, this.n_days])
-        .range([mar, line_width-2*mar])
-
-      // define axes
-      var xLine = this.d3.axisBottom()
-        .scale(xScale)
-        .ticks(0).tickSize(0);
-
-      // draw axes
-      var liney = svg.append("g")
-        .call(xLine)
-        .attr("transform", "translate(0," + 120 + ")")
-        .classed("liney", true)
-
-      // style axes
-      liney.select("path.domain")
-        .attr("id", "timeline-x")
-        .attr("color", "lightgrey")
-        .attr("stroke-width", "3px")
-
-    // timeline events/"buttons"
-      var button_month = svg.append("g")
-      .classed("#btn-month", true)
-      .attr("transform", "translate(0," + 120 + ")")
+      var button_month = time_container.select('svg')
+      .append("g")
+      .attr("transform", "translate(20," + 120 + ")")
       .attr("z-index", 100)
 
     // month points on timeline
-    // TODO: make functions that accept any date and label for annotations
       button_month.selectAll(".button_inner")
-      .data(this.months).enter()
+      .data(this.time_labels).enter()
       .append("circle")
-      .attr("class", function(d,i) { return "button_inner inner_" + d.name + " " + d.name } ) 
-      .attr("r", 3)
-      .attr("cx", function(d) { return xScale(d.day) })
+      .attr("class", function(d,i) { return "button_inner inner_" + d.month_label + "_" + d.year } ) 
+      .attr("r", 5)
+      .attr("cx", function(d) { return self.xScale(d.day_seq) })
       .attr("cy", 0)
-      .attr("stroke", this.button_color)
+      .attr("stroke", "white")
       .attr("stroke-width", "2px")
       .attr("fill", this.button_color)
-      .on('click', function(d, i) {
-        //self.moveTimeline(d); // requires gsap for playback control or some way to cancel playback loop partway
-      })
-      .on('mouseover', function(d, i) {
+      //.on('click', function(d, i) {
+        //self.moveTimeline(d); 
+     // })
+      /* .on('mouseover', function(d, i) {
         self.buttonSelect(d);
       })
       .on('mouseout', function(d, i) {
         self.buttonDeSelect(d);
-      })
+      }) */
 
       // month labels
       button_month.selectAll(".button_name")
-      .data(this.months)
+      .data(this.time_labels)
       .enter()
       .append("text")
-      .attr("class", function(d,i) { return "button_name name_" + d.name + " " + d.name } ) 
-      .attr("x", function(d) { return xScale(d.day)-12 }) // centering on pt
+      .attr("class", function(d,i) { return "button_name name_" + d.month_label + "_" + d.year } ) 
+      .attr("x", function(d) { return self.xScale(d.day_seq)-10 }) // centering on pt
       .attr("y", 25)
-      .attr("stroke", this.button_color)
-      .text(function(d, i) { return d.name })
-      .attr("text-align", "middle")
-      .on('click', function(d, i) {
+      .text(function(d, i) { return d.month_label })
```

Comment:
Is this if you start with a smaller screen size and then make it larger? On the initial draw, all font sizes should be consistent, BUT they scale in the bottom chart when the window is changed. I documented this here (https://github.com/USGS-VIZLAB/gw-conditions/issues/34) and am not sure how to address it. Would you have a change in window size trigger a function that redraws just that chart?

Reply:
Ahh good catch - yes it was. As for when the window is changed, I'm not sure what the best way is to go about it. I think in the past I've either used a fixed font size (instead of em), or set media breaks and defined different font sizes. I don't think either is ideal, though.